composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": ">=5.5",
         "container-interop/container-interop": "^1.1",
-        "zendframework/zend-expressive": "^1.0@rc",
+        "zendframework/zend-expressive": "~1.0.0-dev@dev",
```

Comment:
> It will essentially grab dev-master always

Yes only for this repo. But you can change later :) .

and I can understand the pain. There should be some sort of scripts to do automatic releases and tagging when we have many repos.